test/clone.test.js

```diff
@@ -0,0 +1,18 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const clone = require('../lib/util/clone');
+
+function testCloner(expNode, new_node) {
+  it ('creates a clone of a node', function () {
+    assert.equal(clone(math.parse(expNode)), new_node);
+  });
+}
+
+describe('Clone', function () {
+  const tests = [
+    [1, 1],
+    ['x', 'x']
```

Comment:
I think their we were simplifying the expression and checked the value of the newNode after simplifying. But in this case we are not simplifying the node, we are just cloning. Let me know what you think !